### ISSUE TITLE
Improvement of CSS portion

### ISSUE THREAD

**Issue by noobremon** [yashksaini-coder/BookStream#64]
I would like to improve the design of the website

**Comment by noobremon** [comment]
Please assign this to me @yashksaini-coder

**Comment by yashksaini-coder** [comment]
@noobremon any updates on the issue ??

**Comment by noobremon** [comment]
Yes I have enhanced the CSS part tried to give it attractive interface and now thinking of adding new pages to the website 
@yashksaini-coder
